src/adapters/refract/transformResource.coffee

```diff
@@ -37,18 +37,7 @@ module.exports = (resourceElement, location, options) ->
     attributes = _.dataStructures(resourceElement)
     attributes = if _.isEmpty(attributes) then undefined else attributes[0]
 
-    return [
```

Comment:
The lines above (`attributes = _.dataStructures(resourceElement)`) can be removed? and also the comment above is no longer applicable.